bootcfg/http/ignition.go

```diff
@@ -39,6 +37,20 @@ func ignitionHandler(srv server.Server) ContextHandler {
 			return
 		}
 
+		// Skip rendering if raw Ignition JSON is provided
+		if isIgnition(profile.IgnitionId) {
+			cfg, err := ignition.Parse([]byte(contents))
```

Comment:
After discussion, we'll pass through the contents in all cases and just log any parse errors as warnings.